tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/skyrat/erp_preferences.tsx

```diff
@@ -37,6 +44,13 @@ export const sextoy_pref: FeatureToggle = {
   component: CheckboxInput,
 };
 
+export const sextoy_sounds_pref: FeatureToggle = {
+  name: 'Sex toy sounds',
+  category: 'ERP',
+  description: 'toggles if you will hear sounds from sex toys.',
```

Comment:
I think Caps meant to comment here.
```suggestion
  description: 'Toggles if you will hear sounds from sex toys.',
```